common/src/main/java/com/wynntils/core/consumers/functions/FunctionManager.java

```diff
@@ -236,10 +236,10 @@ public StyledText[] doFormatLines(String templateString) {
             resultBuilder.append(c);
         }
 
-        // Parse color codes before calculating the templates
-        String escapedTemplate = parseColorCodes(resultBuilder.toString());
+        // Calculate templates first to allow for color codes in templates
```

Comment:
@kristofbolyai Are there any unintended side-effects of changing this order?

Reply:
Although I really don't like changing working systems like this, I can reason nothing against this. 

What I don't see is that how or when this is needed? We can have color codes in templates even now?

Reply:
That is the whole purpose of this fix. 

But see my suggestion below to make this clearer, by turning this into an explicit `Color` function.

Reply:
![image](https://github.com/Wynntils/Artemis/assets/49001742/7d85bf59-7604-4a9a-bdb0-9ef2169cdeae)
So why is this needed?

Reply:
(ignore us not reseting the color for that command..)

Reply:
![image](https://github.com/Wynntils/Artemis/assets/57310593/5a50e5fa-0922-4224-8b6a-9773f5618594)

Reply:
You could force them to include & in every iterator set with [&a] or whatever but if there is no side effect of reversing the order, why not?

Reply:
well yeah, i see. I am just always afraid of changing stuff like this..